Hi, in this project, 
1. I want to find out which artists' paintings were acquired after their death, and who were the artists whose acquisitions took the longest time before their death in this museum?

2. I wanted to find out the relationship between the year in which an artist's work was acquired and their age. Whose works in this museum were acquired in the artist's twilight years or even after their death? Whose works were acquired at a young age?
Do artists become more experienced as they age and their work is more likely to be appreciated and acquired?

choose one

The method I used is using the artworks' acquisition Year to minus the artists' Birth Year.

In [3]:
import pandas as pd

In [5]:
pd.options.display.max_rows = 100

In [8]:
artist = pd.read_csv('C:/Users/pc/Desktop/artist_data.csv', delimiter=",", encoding='utf-8').rename(columns={'name':'artist'})
artist

id                  artist  gender      dates  yearOfBirth  \
0     10093  Abakanowicz, Magdalena  Female  born 1930       1930.0   
1         0     Abbey, Edwin Austin    Male  1852–1911       1852.0   
2      2756        Abbott, Berenice  Female  1898–1991       1898.0   
3         1  Abbott, Lemuel Francis    Male  1760–1803       1760.0   
4       622          Abrahams, Ivor    Male  born 1935       1935.0   
...     ...                     ...     ...        ...          ...   
3527  12542         Zorio, Gilberto    Male  born 1944       1944.0   
3528   2186              Zox, Larry    Male  born 1936       1936.0   
3529    621   Zuccarelli, Francesco    Male  1702–1788       1702.0   
3530   2187        Zuloaga, Ignacio    Male  1870–1945       1870.0   
3531   2188         Zyw, Aleksander    Male  1905–1995       1905.0   

      yearOfDeath                    placeOfBirth            placeOfDeath  \
0             NaN                          Polska                     NaN   
1          1911.0     Philadelphia, United States  London, United Kingdom   
2          1991.0      Springfield, United States   Monson, United States   
3          1803.0  Leicestershire, United Kingdom  London, United Kingdom   
4             NaN           Wigan, United Kingdom                     NaN   
...           ...                             ...                     ...   
3527          NaN           Andorno Micca, Italia                     NaN   
3528          NaN       Des Moines, United States                     NaN   
3529       1788.0                          Italia                 Firenze   
3530       1945.0                          España                  España   
3531       1995.0                          Polska                     NaN   

                                                    url  
0     http://www.tate.org.uk/art/artists/magdalena-a...  
1     http://www.tate.org.uk/art/artists/edwin-austi...  
2     http://www.tate.org.uk/art/artists/berenice-ab...  
3     http://www.tate.org.uk/art/artists/lemuel-fran...  
4     http://www.tate.org.uk/art/artists/ivor-abraha...  
...                                                 ...  
3527  http://www.tate.org.uk/art/artists/gilberto-zo...  
3528  http://www.tate.org.uk/art/artists/larry-zox-2186  
3529  http://www.tate.org.uk/art/artists/francesco-z...  
3530  http://www.tate.org.uk/art/artists/ignacio-zul...  
3531  http://www.tate.org.uk/art/artists/aleksander-...  

[3532 rows x 9 columns]

HERE I renamed the column 'name' to 'artist' in order to find a same join name with the other csv.

In [9]:
artwork = pd.read_csv('C:/Users/pc/Desktop/artwork_data.csv', low_memory=False, delimiter=',',encoding='utf-8')
artwork

id accession_number              artist artistRole  artistId  \
0        1035           A00001       Blake, Robert     artist        38   
1        1036           A00002       Blake, Robert     artist        38   
2        1037           A00003       Blake, Robert     artist        38   
3        1038           A00004       Blake, Robert     artist        38   
4        1039           A00005      Blake, William     artist        39   
...       ...              ...                 ...        ...       ...   
69196  122960           T13865  P-Orridge, Genesis     artist     16646   
69197  122961           T13866  P-Orridge, Genesis     artist     16646   
69198  121181           T13867        Hatoum, Mona     artist      2365   
69199  112306           T13868       Creed, Martin     artist      2760   
69200  127035           T13869   Brunias, Agostino     artist      3824   

                                                   title  \
0      A Figure Bowing before a Seated Old Man with h...   
1      Two Drawings of Frightened Figures, Probably f...   
2      The Preaching of Warning. Verso: An Old Man En...   
3         Six Drawings of Figures with Outstretched Arms   
4      The Circle of the Lustful: Francesca da Rimini...   
...                                                  ...   
69196                        Larvae (from Tampax Romana)   
69197                   Living Womb (from Tampax Romana)   
69198                                      Present Tense   
69199          Work No. 227: The lights going on and off   
69200                   Dancing Scene in the West Indies   

                     dateText  \
0              date not known   
1              date not known   
2                     ?c.1785   
3              date not known   
4      1826–7, reprinted 1892   
...                       ...   
69196                    1975   
69197                    1976   
69198                    1996   
69199                    2000   
69200                 1764–96   

                                                  medium  \
0      Watercolour, ink, chalk and graphite on paper....   
1                                      Graphite on paper   
2            Graphite on paper. Verso: graphite on paper   
3                                      Graphite on paper   
4                                Line engraving on paper   
...                                                  ...   
69196   Perspex, Wood, hairpiece, tampon and human blood   
69197  Wood, Perspex, plastic, photograph on paper, t...   
69198                               Soap and glass beads   
69199                                   Gallery lighting   
69200                                Oil paint on canvas   

                                              creditLine  year  \
0                    Presented by Mrs John Richmond 1922   NaN   
1                    Presented by Mrs John Richmond 1922   NaN   
2                    Presented by Mrs John Richmond 1922  1785   
3                    Presented by Mrs John Richmond 1922   NaN   
4      Purchased with the assistance of a special gra...  1826   
...                                                  ...   ...   
69196                 Transferred from Tate Archive 2012  1975   
69197                 Transferred from Tate Archive 2012  1976   
69198                     Presented by Tate Members 2013  1996   
69199  Purchased with funds provided by Tate Members,...  2000   
69200  Purchased with assistance from Tate Patrons an...  1764   

       acquisitionYear                           dimensions width height  \
0               1922.0                support: 394 x 419 mm   394    419   
1               1922.0                support: 311 x 213 mm   311    213   
2               1922.0                support: 343 x 467 mm   343    467   
3               1922.0                support: 318 x 394 mm   318    394   
4               1919.0                  image: 243 x 335 mm   243    335   
...                ...

Then merge them together.

In [10]:
merged = pd.merge(artist, artwork, on='artist')
merged

id_x                  artist  gender      dates  yearOfBirth  \
0      10093  Abakanowicz, Magdalena  Female  born 1930       1930.0   
1      10093  Abakanowicz, Magdalena  Female  born 1930       1930.0   
2      10093  Abakanowicz, Magdalena  Female  born 1930       1930.0   
3      10093  Abakanowicz, Magdalena  Female  born 1930       1930.0   
4          0     Abbey, Edwin Austin    Male  1852–1911       1852.0   
...      ...                     ...     ...        ...          ...   
69308  12542         Zorio, Gilberto    Male  born 1944       1944.0   
69309   2186              Zox, Larry    Male  born 1936       1936.0   
69310    621   Zuccarelli, Francesco    Male  1702–1788       1702.0   
69311   2187        Zuloaga, Ignacio    Male  1870–1945       1870.0   
69312   2188         Zyw, Aleksander    Male  1905–1995       1905.0   

       yearOfDeath                 placeOfBirth            placeOfDeath  \
0              NaN                       Polska                     NaN   
1              NaN                       Polska                     NaN   
2              NaN                       Polska                     NaN   
3              NaN                       Polska                     NaN   
4           1911.0  Philadelphia, United States  London, United Kingdom   
...            ...                          ...                     ...   
69308          NaN        Andorno Micca, Italia                     NaN   
69309          NaN    Des Moines, United States                     NaN   
69310       1788.0                       Italia                 Firenze   
69311       1945.0                       España                  España   
69312       1995.0                       Polska                     NaN   

                                                   url_x    id_y  ...  \
0      http://www.tate.org.uk/art/artists/magdalena-a...  102939  ...   
1      http://www.tate.org.uk/art/artists/magdalena-a...  102938  ...   
2      http://www.tate.org.uk/art/artists/magdalena-a...  103642  ...   
3      http://www.tate.org.uk/art/artists/magdalena-a...  103644  ...   
4      http://www.tate.org.uk/art/artists/edwin-austi...     311  ...   
...                                                  ...     ...  ...   
69308  http://www.tate.org.uk/art/artists/gilberto-zo...  103399  ...   
69309  http://www.tate.org.uk/art/artists/larry-zox-2186   16531  ...   
69310  http://www.tate.org.uk/art/artists/francesco-z...   16532  ...   
69311  http://www.tate.org.uk/art/artists/ignacio-zul...   16533  ...   
69312  http://www.tate.org.uk/art/artists/aleksander-...   16534  ...   

      acquisitionYear                                         dimensions  \
0              2009.0                Overall display dimensions variable   
1              2009.0          Display dimensions: 4050 x 3820 x 4000 mm   
2              2009.0          Display dimensions: 4010 x 2900 x 3700 mm   
3              2009.0                   object, each: 660 x 580 x 680 mm   
4              1924.0                              support: 251 x 295 mm   
...               ...                                                ...   
69308          2009.0  object: 125 x 1113 x 190 mm image: 340 x 287 m...   
69309          1968.0                            support: 2235 x 2489 mm   
69310          1985.0      support: 1264 x 1572 mm frame: 1497 x 1800 mm   
69311          1923.0     support: 292 x 406 mm frame: 470 x 585 x 93 mm   
69312          1962.0     frame: 934 x 662 x 28 mm support: 920 x 649 mm   

       width height  depth units     inscription  \
0        NaN    NaN    NaN   NaN             NaN   
1        NaN    NaN    NaN   NaN             NaN   
2        NaN    NaN    NaN   NaN             NaN   
3        NaN    NaN    NaN   NaN             NaN   
4        251    295    NaN    mm             NaN   
...      ...    ...    ...   ...             ...   
69308    125   1113  190.0    mm             NaN   
69309   2235   2489    NaN    mm  da

We got the merged dataset, so we may visualize the trend first.

In [11]:
import seaborn as sns
import matplotlib.pyplot as plt
sns.set_theme()
sns.relplot(data=merged, x='yearOfBrith', y='acquisitionYear')
plt.show()

ValueError: Could not interpret value `yearOfBrith` for parameter `x`

With the development of technology, more art pieces were acquired around 2000. 

This could be for a number of reasons, such as the development of technology leading to an increase in the standard of authenticating paintings, making it easier to distinguish the real from the fake, and merchants becoming more willing to invest in art; or it could be that government promotion, the increase in art academies and people's aesthetic appreciation of art has led to more paintings being acquired; or it could be that the accelerated process of globalization and the development of the internet has made art more accessible, etc. The reasons behind this phenomenon of this museum are worth pondering.

We can calculate that.

In [ ]:
timelength = merged['acquisitionYear'] - merged['yearOfBirth']
timelength = pd.DataFrame(timelength)
timelength.columns=['timegap']
timelength

In [154]:
df1 = pd.concat([merged['artist'], timelength['timegap']],axis=1, keys=['artist','timegap'])
df1

artist  timegap
0      Abakanowicz, Magdalena      NaN
1      Abakanowicz, Magdalena      NaN
2      Abakanowicz, Magdalena      NaN
3      Abakanowicz, Magdalena      NaN
4         Abbey, Edwin Austin     13.0
...                       ...      ...
69308         Zorio, Gilberto      NaN
69309              Zox, Larry      NaN
69310   Zuccarelli, Francesco    197.0
69311        Zuloaga, Ignacio    -22.0
69312         Zyw, Aleksander    -33.0

[69313 rows x 2 columns]

In [167]:
gkk = df1.groupby(['artist'])['timegap'].sum().reset_index().rename(columns={'timegap':'total_year'})
gkk

artist  total_year
0     Abakanowicz, Magdalena         0.0
1        Abbey, Edwin Austin        78.0
2           Abbott, Berenice        57.0
3     Abbott, Lemuel Francis       247.0
4             Abrahams, Ivor         0.0
...                      ...         ...
3327        Zuloaga, Ignacio       -22.0
3328         Zyw, Aleksander       -33.0
3329         di Suvero, Mark         0.0
3330            van Elk, Ger         0.0
3331       Štyrský, Jindrich       130.0

[3332 rows x 2 columns]

In [166]:
ggg = df1.groupby(['artist'])['timegap'].count().reset_index()
ggg = ggg.rename(columns={'timegap':'piece_num'})
ggg

artist  piece_num
0     Abakanowicz, Magdalena          0
1        Abbey, Edwin Austin          6
2           Abbott, Berenice          3
3     Abbott, Lemuel Francis          2
4             Abrahams, Ivor          0
...                      ...        ...
3327        Zuloaga, Ignacio          1
3328         Zyw, Aleksander          1
3329         di Suvero, Mark          0
3330            van Elk, Ger          0
3331       Štyrský, Jindrich          2

[3332 rows x 2 columns]

In [243]:
merged2 = pd.merge(gkk, ggg, on='artist')
merged2['ave_gap'] = merged2['total_year'] / merged2['piece_num']
merged2['rating_rank'] = merged2['ave_gap'].rank(ascending=0)
merged2 = merged2.set_index('rating_rank')
merged2 = merged2.sort_index()
merged2.head(100)

artist  total_year  piece_num  \
rating_rank                                                                     
1.0                                       Eworth, Hans      1200.0          3   
2.0                                      Oliver, Isaac      1520.0          4   
3.0                                 Rubens, Peter Paul       369.0          1   
4.0                               Bacon, Sir Nathaniel       368.0          1   
5.0                        British School 16th century      1451.0          4   
6.0                                    Colone, Adam de       361.0          1   
7.0                                      Gower, George       712.0          2   
8.0                              Van Dyck, Sir Anthony      1059.0          3   
9.0                                 Segar, Sir William       350.0          1   
10.0                            Gheeraerts, Marcus, II      1739.0          5   
11.0                                   Van Somer, Paul       340.0          1   
12.0                                  Jackson, Gilbert       339.0          1   
13.0                                     Peake, Robert       672.0          2   
14.0                               Keirincx, Alexander       334.0          1   
15.0                                    Cleyn, Francis       332.0          1   
16.0                                      Bettes, John       327.0          1   
17.0                                   Wissing, Willem       319.0          1   
18.0                                   Anderton, Henry       318.0          1   
19.0                                       Hayls, John       316.0          1   
20.0                                   Johnson, Thomas       312.0          1   
21.0                                       Souch, John       310.0          1   
22.0                                       Beale, Mary       917.0          3   
23.0                                Des Granges, David       301.0          1   
24.5                                     Manby, Thomas       298.0          1   
24.5                                   Verelst, Harman       298.0          1   
26.0                                     Bower, Edward       295.0          1   
27.0                                    Soest, Gilbert       589.0          2   
28.0                                  Hook, Dr Richard       294.0          1   
29.0                                   Barlow, Francis     31891.0        109   
30.0                                   Dobson, William       874.0          3   
31.0                                    Du Bois, Simon       290.0          1   
32.0                                      Gyles, Henry       288.0          1   
33.0                              Wright, John Michael       859.0          3   
34.0                       British School 17th century      6561.0         23   
35.0                                       Riley, John       570.0          2   
36.0                                     Fuller, Isaac       283.0          1   
37.5                                   Siberechts, Jan       562.0          2   
37.5                                      Gibbs, Henry       281.0          1   
39.0                                Gennari, Benedetto       279.0          1   
40.0                                  Sailmaker, Isaac       276.0          1   
41.0                                Johnson, Cornelius      1926.0          7   
42.0                                Cradock, Marmaduke       274.0          1   
43.0                         Mytens, Daniel, the Elder       272.0          1   
44.0                                   Huysmans, Jacob       271.0          1   
45.0                                    Place, Francis      1345.0          5   
46.5                                  Borseller, Peter       266.0          1   
46.5                                 Le Piper, Francis      1064.0          4   
48.0                                   Lely, Sir Peter      2125.0          8  

In [236]:
print(merged2.dtypes)

artist         object
total_year    float64
piece_num       int64
ave_gap       float64
dtype: object


In [249]:
after_death = merged2['ave_gap'][merged2['ave_gap'] >= 0].count()
print(after_death)

1248


In [248]:
before_death = merged2['ave_gap'][merged2['ave_gap'] < 0].count()
print(before_death)

907


From this calculation, it is not difficult to see that 1248 artists became famous, while 907 artists had already become famous before they passed away.

There are some other experiments I did. Not important.

In [42]:
merged[merged['gender'] == 'Male']

id                   name gender      dates  yearOfBirth  \
2       622         Abrahams, Ivor   Male  born 1935       1935.0   
3      2606                Absalon   Male  1964–1993       1964.0   
5       623          Acconci, Vito   Male  born 1940       1940.0   
6       624         Ackling, Roger   Male  1947–2014       1947.0   
7       625        Ackroyd, Norman   Male  born 1938       1938.0   
...     ...                    ...    ...        ...          ...   
3341  12542        Zorio, Gilberto   Male  born 1944       1944.0   
3342   2186             Zox, Larry   Male  born 1936       1936.0   
3343    621  Zuccarelli, Francesco   Male  1702–1788       1702.0   
3344   2187       Zuloaga, Ignacio   Male  1870–1945       1870.0   
3345   2188        Zyw, Aleksander   Male  1905–1995       1905.0   

      yearOfDeath               placeOfBirth   placeOfDeath  \
2             NaN      Wigan, United Kingdom            NaN   
3          1993.0    Tel Aviv-Yafo, Yisra'el  Paris, France   
5             NaN    New York, United States            NaN   
6          2014.0  Isleworth, United Kingdom            NaN   
7             NaN      Leeds, United Kingdom            NaN   
...           ...                        ...            ...   
3341          NaN      Andorno Micca, Italia            NaN   
3342          NaN  Des Moines, United States            NaN   
3343       1788.0                     Italia        Firenze   
3344       1945.0                     España         España   
3345       1995.0                     Polska            NaN   

                                                  url_x accession_number  ...  \
2     http://www.tate.org.uk/art/artists/ivor-abraha...           A01034  ...   
3       http://www.tate.org.uk/art/artists/absalon-2606           N02650  ...   
5     http://www.tate.org.uk/art/artists/vito-acconc...           A01035  ...   
6     http://www.tate.org.uk/art/artists/roger-ackli...           A01036  ...   
7     http://www.tate.org.uk/art/artists/norman-ackr...           A01037  ...   
...                                                 ...              ...  ...   
3341  http://www.tate.org.uk/art/artists/gilberto-zo...           T00925  ...   
3342  http://www.tate.org.uk/art/artists/larry-zox-2186           N04441  ...   
3343  http://www.tate.org.uk/art/artists/francesco-z...           T04112  ...   
3344  http://www.tate.org.uk/art/artists/ignacio-zul...           P01009  ...   
3345  http://www.tate.org.uk/art/artists/aleksander-...           N03905  ...   

     acquisitionYear                                       dimensions  width  \
2             1961.0                                              NaN    NaN   
3             1910.0  support: 324 x 505 mm frame: 570 x 747 x 100 mm    324   
5             1961.0                                              NaN    NaN   
6             1961.0                                              NaN    NaN   
7             1961.0                                              NaN    NaN   
...              ...                                              ...    ...   
3341          1967.0   support: 406 x 498 mm frame: 580 x 676 x 74 mm    406   
3342          1929.0                            support: 273 x 378 mm    273   
3343          1985.0                       object: 815 x 195 x 146 mm    815   
3344          1918.0                              image: 133 x 406 mm    133   
3345          1924.0                            support: 250 x 370 mm    250   

     height  depth units     inscription thumbnailCopyright  \
2       NaN    NaN   NaN             NaN                NaN   
3       505    NaN    mm             NaN                NaN   
5       NaN    NaN   NaN             NaN                NaN   
6       NaN    NaN   NaN             NaN                NaN   
7       NaN    NaN   NaN             NaN                NaN   
...     ...    ...   ...             ...                ...   
3341    498    NaN    mm  date inscribed        

In [43]:
merged[merged['gender'] == 'Female']

id                    name  gender      dates  yearOfBirth  \
0     10093  Abakanowicz, Magdalena  Female  born 1930       1930.0   
1      2756        Abbott, Berenice  Female  1898–1991       1898.0   
4      9550             Abts, Tomma  Female  born 1967       1967.0   
13     2608           Adshead, Mary  Female  1904–1995       1904.0   
16      633            Agar, Eileen  Female  1899–1991       1899.0   
...     ...                     ...     ...        ...          ...   
3317   2386         Yass, Catherine  Female  born 1963       1963.0   
3320   2179          Yhap, Laetitia  Female  born 1941       1941.0   
3321   8297            Young, Carey  Female  born 1970       1970.0   
3329  14306       Zaccagnini, Carla  Female  born 1973       1973.0   
3338   2395          Zittel, Andrea  Female  born 1965       1965.0   

      yearOfDeath                  placeOfBirth               placeOfDeath  \
0             NaN                        Polska                        NaN   
1          1991.0    Springfield, United States      Monson, United States   
4             NaN             Kiel, Deutschland                        NaN   
13         1995.0        London, United Kingdom  Hampstead, United Kingdom   
16         1991.0           Ayacucho, Argentina     London, United Kingdom   
...           ...                           ...                        ...   
3317          NaN        London, United Kingdom                        NaN   
3320          NaN  Saint Albans, United Kingdom                        NaN   
3321          NaN                Lusaka, Zambia                        NaN   
3329          NaN       Buenos Aires, Argentina                        NaN   
3338          NaN      Sausalito, United States                        NaN   

                                                  url_x accession_number  ...  \
0     http://www.tate.org.uk/art/artists/magdalena-a...           P02841  ...   
1     http://www.tate.org.uk/art/artists/berenice-ab...           T04095  ...   
4     http://www.tate.org.uk/art/artists/tomma-abts-...           N01611  ...   
13    http://www.tate.org.uk/art/artists/mary-adshea...           N02654  ...   
16    http://www.tate.org.uk/art/artists/eileen-agar...           T00312  ...   
...                                                 ...              ...  ...   
3317  http://www.tate.org.uk/art/artists/catherine-y...           P02286  ...   
3320  http://www.tate.org.uk/art/artists/laetitia-yh...           N02290  ...   
3321  http://www.tate.org.uk/art/artists/carey-young...           P04556  ...   
3329  http://www.tate.org.uk/art/artists/carla-zacca...           P05221  ...   
3338  http://www.tate.org.uk/art/artists/andrea-zitt...           P02295  ...   

     acquisitionYear                                      dimensions  width  \
0             1986.0                             image: 225 x 251 mm    225   
1             1985.0                             image: 140 x 219 mm    140   
4             1892.0                           support: 740 x 914 mm    740   
13            1910.0  support: 137 x 190 mm frame: 253 x 305 x 65 mm    137   
16            1959.0  support: 305 x 216 mm frame: 393 x 294 x 53 mm    305   
...              ...                                             ...    ...   
3317          1976.0                             image: 440 x 347 mm    440   
3320          1908.0                           support: 295 x 394 mm    295   
3321          1975.0                             image: 776 x 567 mm    776   
3329          1975.0                             image: 952 x 654 mm    952   
3338          1976.0                             image: 397 x 581 mm    397   

     height depth units     inscription  \
0       251   NaN    mm             NaN   
1       219   NaN    mm             NaN   
4       914   NaN    mm             NaN   
13      190   NaN    mm             NaN   
16      216   NaN    mm             NaN   
...     ...   ...   ...             ...

In [44]:
merged[merged['placeOfBirth'] == merged['placeOfDeath']]

id                      name gender      dates  yearOfBirth  \
9      626      Adams, Harry William   Male  1868–1947       1868.0   
29    2470        Alexander, William   Male  1767–1816       1767.0   
32       3       Alken, Henry Thomas   Male  1785–1851       1785.0   
33    2471     Alken, Samuel, Senior   Male  1756–1815       1756.0   
36       4        Allan, Sir William   Male  1782–1850       1782.0   
...    ...                       ...    ...        ...          ...   
3296   615  Wright, Joseph, of Derby   Male  1734–1797       1734.0   
3301  2174         Wyckaert, Maurice   Male  1923–1996       1923.0   
3325  2181          Yuon, Konstantin   Male  1875–1958       1875.0   
3326  2182       Yvaral, Jean-Pierre   Male  1934–2002       1934.0   
3344  2187          Zuloaga, Ignacio   Male  1870–1945       1870.0   

      yearOfDeath               placeOfBirth               placeOfDeath  \
9          1947.0  Worcester, United Kingdom  Worcester, United Kingdom   
29         1816.0  Maidstone, United Kingdom  Maidstone, United Kingdom   
32         1851.0     London, United Kingdom     London, United Kingdom   
33         1815.0     London, United Kingdom     London, United Kingdom   
36         1850.0  Edinburgh, United Kingdom  Edinburgh, United Kingdom   
...           ...                        ...                        ...   
3296       1797.0      Derby, United Kingdom      Derby, United Kingdom   
3301       1996.0          Bruxelles, België          Bruxelles, België   
3325       1958.0            Moskva, Rossiya            Moskva, Rossiya   
3326       2002.0              Paris, France              Paris, France   
3344       1945.0                     España                     España   

                                                  url_x accession_number  ...  \
9     http://www.tate.org.uk/art/artists/harry-willi...           P03138  ...   
29    http://www.tate.org.uk/art/artists/william-ale...           N03577  ...   
32    http://www.tate.org.uk/art/artists/henry-thoma...           T07799  ...   
33    http://www.tate.org.uk/art/artists/samuel-alke...           T03301  ...   
36    http://www.tate.org.uk/art/artists/sir-william...           T12038  ...   
...                                                 ...              ...  ...   
3296  http://www.tate.org.uk/art/artists/joseph-wrig...           T03229  ...   
3301  http://www.tate.org.uk/art/artists/maurice-wyc...           N02125  ...   
3325  http://www.tate.org.uk/art/artists/konstantin-...           T00597  ...   
3326  http://www.tate.org.uk/art/artists/jean-pierre...           P06705  ...   
3344  http://www.tate.org.uk/art/artists/ignacio-zul...           P01009  ...   

     acquisitionYear                                         dimensions  \
9             1976.0                                image: 298 x 298 mm   
29            1921.0                              support: 241 x 349 mm   
32            2001.0  support: 650 x 810 x 19 mm frame: 743 x 904 x ...   
33            1981.0                              support: 705 x 910 mm   
36            2005.0  unconfirmed: 457 x 559 mm frame: 675 x 765 x 8...   
...              ...                                                ...   
3296          1981.0                              support: 521 x 622 mm   
3301          1907.0                              support: 241 x 292 mm   
3325          1963.0                         object: 495 x 330 x 273 mm   
3326          1976.0                                image: 483 x 733 mm   
3344          1918.0                                image: 133 x 406 mm   

      width height  depth units     inscription  \
9       298    298    NaN    mm  date inscribed   
29      241    349    NaN    mm             NaN   
32      650    810   19.0    mm             NaN   
33      705    910    NaN    mm  date inscribed   
36      457    559    NaN    mm             NaN   
...     ...    ...    ...   ...             ...   
3296    521    622    N